src/test/java/br/com/akato/comissaovenda/CalculadoraDeComissaoTest.java

```diff
@@ -12,6 +12,7 @@ public void criaCalculadora(){
 		inicializaObjetos();
 	}
 	
+	
```

Comment:
Usar herança aqui é meio forçado. Você não poderia apenas fazer uso do builder? Ele seria um componente que você usaria. Ele não precisa ser o seu pai